utils/determine-turbo-hit-or-miss.js

```diff
@@ -1,30 +1,35 @@
 const fs = require('fs-extra');
 const path = require('path');
 
-async function main() {
-  const rootDir = path.join(__dirname, '..');
-  const turboRunDir = path.join(rootDir, '.turbo/runs');
+async function main(turboRunDirectory, turboRunDirectoryParent) {
+  const turboRunDir = path.join(turboRunDirectoryParent, turboRunDirectory);
   const turboRunFiles = await fs.readdir(turboRunDir);
 
-  turboRunFiles.forEach(async fileName => {
-    const runFile = path.join(turboRunDir, fileName);
-    const runData = await fs.readJson(runFile);
-    const tasksReports = runData.tasks || [];
+  let missCount = 0;
 
-    const missCount = tasksReports.reduce((total, taskData) => {
-      if (taskData.cache.status === 'MISS') {
-        return total + 1;
-      }
+  await Promise.all(
+    turboRunFiles.map(async fileName => {
```

Comment:
Do you know how many files are in the `.turbo/runs` directory? I've it's dozens, then cool, but if it's hundreds or more, I have see issues with degraded performance and used something like https://www.npmjs.com/package/p-limit to limit the "concurrency".

Reply:
@cb1kenobi currently only ever 1, but there was a suggestion somewhere in Slack that we might have multiple if we ever changed our test matrix to run `turbo` several times in on run. Then it would have 1 file per invocation of `turbo --summarize`